### PR TITLE
chore: add help link for wrong npm version

### PR DESCRIPTION
**Pull request by HuihuiWu-Microsoft**
image

### CODE REVIEW

File: docs/fx-core/spfx-help.md [hunk 1]
```diff
@@ -1,72 +1,99 @@
...
+
+```sh
+yo @microsoft/sharepoint 
+```
+
+SPFx scaffolding could fail due to unsuccessful execution of above command.
+
+#### _"Failed to run command: " yo @microsoft/sharepoint ..." , 'yo' is not recognized as an internal or external command"_
+
+If your os system is Windows,
```
Comment (xu-hong) [inline docs/fx-core/spfx-help.md:87]: nit: OS system

File: docs/fx-core/spfx-help.md [hunk 2]
```diff
@@ -1,72 +1,99 @@
...
+
+1. Check your npm and Node.js version. [Supported Node.js versions for SPFx v1.14.0](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/set-up-your-development-environment#install-nodejs) are:
+
+- Node.js v12.13.0+ (_Erbium_)
+- Node.js v14.15.0+ (_Fermium_)
+Corresponding npm version is v6.x. Please make sure you have the right version installed for both npm and Node.js.
+
+2. Check executable files for `yo` exist after package installation at the correct path under _HOME\\.fx\bin\spfx_
+
+If your os system is MacOS or Linux,
```
Comment (xu-hong) [inline docs/fx-core/spfx-help.md:97]: same

File: docs/fx-core/spfx-help.md [hunk 3]
```diff
@@ -1,72 +1,99 @@
...
+
+1. Check executable files for `yo` exist after package installation at the correct path under _HOME\\.fx\bin\spfx\node_modules\\.bin_
```
Comment (xu-hong) [inline docs/fx-core/spfx-help.md:99]: `_HOME/.fx/bin/spfx/node_modules/.bin_`?

Comment (HuihuiWu-Microsoft) [reply docs/fx-core/spfx-help.md:99]: The second slash is for escape '.'
image

File: docs/fx-core/spfx-help.md (hunk 1 above)
Comment (HuihuiWu-Microsoft) [reply docs/fx-core/spfx-help.md]: Updated

File: docs/fx-core/spfx-help.md (hunk 2 above)
Comment (HuihuiWu-Microsoft) [reply docs/fx-core/spfx-help.md]: Updated

File: docs/fx-core/spfx-help.md (hunk 3 above)
Comment (xu-hong) [reply docs/fx-core/spfx-help.md:99]: I meant for MacOS or Linux, shouldn't it be forward slash `/`?

Comment (HuihuiWu-Microsoft) [reply docs/fx-core/spfx-help.md]: image

**Review by nliu-ms — APPROVED** [review]

**Pull request merged**